backend/app/routers/helpers/helpers.py

```diff
@@ -0,0 +1,67 @@
+from typing import Tuple, Dict
```

Comment:
"Helpers" is an anti-pattern.

https://softwareengineering.stackexchange.com/questions/422907/is-putting-general-use-functions-in-a-helpers-file-an-anti-pattern-or-code-sme

Reply:
It seems like this logic could migrate to a class that's responsible for manipulating the storage.

Reply:
Yes, I removed this file and created `StorageManager` to handle operations related to storage manipulation.